persistence/gorm/gorm.go

```diff
@@ -87,15 +93,17 @@
 	}
 
 	// After successful DB initialization, migrate the schema
-	// Migrate User
-	if err = g.db.AutoMigrate(&auth.User{}); err != nil {
-		err = fmt.Errorf("error during auto-migration: %w", err)
-		return
+	var types = []interface{}{
```

Comment:
We could also put everything directly into `AutoMigrate` (variadic function). I can do it in the certificate API PR though.